bidding_auction_services_multi_seller_auctions.md

```diff
@@ -306,6 +306,21 @@ The updated definition for AuctionResult will be as follows:
 _Note: Additional metadata will be sent back to the client for reporting (URL generation and report URL ping) after
 the top-level auction on the browser. The API changes will be published at a later date in a different explainer._
 
+
+#### Ad Tech Specifications
+##### Component Seller Ad Server
+The seller's ad server has to orchestrate the flow to conduct the component auction which includes - 
+1. Sending a SelectAdRequest to SellerFrontEnd service with a B&A ciphertext and an auctionConfig as described in the [API changes][28].
+
+_Note: For a component auction, the auctionConfig must contain a ‘topLevelSeller’ field with the domain of the top level seller which will call runAdAuction on the device._
+
+2. Including the encrypted AuctionResult responses (returned from the SelectAd RPC in step 1) in the response to the browser. 
+3. Attaching the response header containing the base64url encoded SHA-256 hash of the response blob for each response blob sent back to the [browser][29]. This is required by the browser to verify that it witnessed a fetch request to the seller’s origin with "adAuctionHeaders: true" that included an Ad-Auction-Result header with hash of response_blob in the response.
```

Comment:
3. Attaches [SHA-256](https://en.wikipedia.org/wiki/SHA-2) hash for each response blob included in an HTTP response header (Ad-Auction-Result header) back to the [browser][29]. 
_Note: This is required by the browser to ensure that the encrypted AuctionResult response is in fact coming from the correct seller origin. Refer [here](https://github.com/WICG/turtledove/blob/main/FLEDGE_browser_bidding_and_auction_API.md#step-4-complete-auction-in-browser) for more details_. 

Following is an example of Ad-Auction-Result header.